clang/lib/Sema/SemaDeclCXX.cpp

```diff
@@ -6036,6 +6036,57 @@ static bool canPassInRegisters(Sema &S, CXXRecordDecl *D,
   return HasNonDeletedCopyOrMove;
 }
 
+/// Add [[gsl::Owner]] and [[gsl::Pointer]] attributes for std:: types.
+/// \pre Record->getIdentifiers() is non-null
+void Sema::addOwnerPointerAttribute(CXXRecordDecl *Record) {
+
+  static llvm::StringSet<> StdOwners{
```

Comment:
For some reason LLVM folks do not really like static initializers. I windows if we could make this a const member of Sema and init it in the ctor. We could also wait with this change until the reviewers actually want it.

Reply:
Note that this is not a static initializer in the sense that it runs before main - it runs on first entry to the function. I would wait for the review.
I actually would love to make this a constexpr container, but there seems to be no constexpr map in std/llvm yet.